src/pages/Groups/Group4/Screen1.tsx

```diff
@@ -1,7 +1,230 @@
+`
+const ForgetPasswordLabel = styled.text`
+  float: left;
+  font-size: 10px;
+  padding: 0px 0px 4px 0px;
+  font-family: Inter;
+  font-style: normal;
+  font-weight: 200;
+`
+const Logo = styled.img`
+  width: 70%;
+  max-width: 255px;
+  max-height: 255px;
+`
+const ForgetPwButton = styled.div`
+  text-align: left;
+  text-decoration: underline;
+  color: #72ba75;
+  font-size: 13px;
+  .ant-btn {
+    letter-spacing: 0em;
+    text-align: center;
+    font-weight: 200;
+    color: #72ba75;
+  }
+  .ant-btn-primary {
+    background-color: #72ba75;
+    border-color: #72ba75;
+    width: 100%;
+    margin-top: 10px;
+  }
+  .ant-btn-primary:focus {
+    background-color: #72ba75;
+    border-color: #72ba75;
+    background: #72ba75;
+    border-color: #72ba75;
+    font-weight: bold;
+  }
+  .ant-btn-primary:hover {
+    background-color: #72ba75;
+    border-color: #72ba75;
+    background: #72ba75;
+    border-color: #72ba75;
+    font-weight: bold;
+  }
+  .ant-btn-default: {
+    height: min-content;
+    margin: 10px;
+    float: left;
+    border-radius: 20px;
+  }
+`
+const PostButton = styled.div`
+  text-align: center;
+  .ant-btn {
+    font-size: 17px;
+    letter-spacing: 0em;
+    text-align: center;
+    font-weight: 200;
+  }
+  .ant-btn-primary {
+    background-color: #72ba75;
+    border-color: #72ba75;
+    width: 100%;
+    margin-top: 10px;
+  }
+  .ant-btn-primary:focus {
+    background-color: #72ba75;
+    border-color: #72ba75;
+    background: #72ba75;
+    border-color: #72ba75;
+  }
+  .ant-btn-primary:hover {
+    background-color: #72ba75;
+    border-color: #72ba75;
+    background: #72ba75;
+    border-color: #72ba75;
+  }
+  .ant-btn-default: {
+    height: min-content;
+    margin: 10px;
+    float: left;
+    border-radius: 20px;
+  }
+`
+const AlertGroup = styled.div`
+  margin: 0px;
+`
+const StyledUsernameInput = styled.div`
+  .ant-input {
+    border-radius: 20px;
+    font-size: 20px;
+    font-weight: 200;
+  }
+`
+const StyledPasswordInput = styled.div`
+  .ant-input-password {
+    border-radius: 20px;
+    font-size: 20px;
+  }
+  .ant-input {
+    border-radius: 15px;
+    font-size: 20px;
+    font-weight: 200;
+  }
+`
+
+const ButtonLabel = styled.div`
+  width: 100%;
+  white-space: break-spaces;
+  overflow: hidden;
+`
+
+const StyledButtonContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 70px;
+  column-gap: 6%;
+`
+
+const LoginPage = () => {
+  return (
+    <ContainerDiv>
+      {/* {isLoading && <LoadingSpin />} */}
+      <LoginContainer>
+        <Logo src={logo} />
+        <br />
+        <InputTextLabel>Username: </InputTextLabel>
+        <StyledUsernameInput>
+          <Input
+            type="text"
+            placeholder="Matric Number"
+            // value={username}
+            onChange={(e) => {
+              const newUsername = e.target.value
+              // setUsername(newUsername.toUpperCase())
```

Comment:
remove unused comments...

Reply:
Fixed in commit c9be4feb05ed8549d3f246c1f6ceaaece59d06f8